metadata-ingestion/src/datahub/ingestion/source/looker/looker_lib_wrapper.py

```diff
@@ -205,8 +205,9 @@ def lookml_model_explore(self, model: str, explore_name: str) -> LookmlModelExpl
     def folder_ancestors(
         self,
         folder_id: str,
-        fields: Union[str, List[str]] = ["id", "name", "parent_id"],
+        fields: Optional[Union[str, List[str]]] = None,
```

Comment:
While in practice this is not a breaking change, I prefer to keep the default value in the signature than inside the implementation. Because knowing the default value is relevant for the caller.

Reply:
mutable args can lead to bugs https://docs.python-guide.org/writing/gotchas/ This is a well documented problem.